client/styles/sc-text-heading-styles.html

```diff
@@ -222,6 +222,17 @@
         padding: 0;
       }
 
+      #segmented_text_content .hgroup:after {
+        content: "NOTE: This is a draft. The proofreading is not complete.";
```

Comment:
Will this be internationalized? Why put it in :after?

Reply:
Basically just what Sujato gave me to put in and no, it won't be internationalized: It's only on English pages anyway and only temporarily.